src/cmd/src/lib.rs

```diff
@@ -21,4 +21,3 @@ pub mod frontend;
 pub mod metasrv;
 pub mod options;
 pub mod standalone;
-mod toml_loader;
```

Comment:
Is there a dependency crate we can remove too?

Reply:
Because `config-rs` already did the same thing of paring toml file, I think we can remove the duplicate code that doesn't use it.

Reply:
Can the `toml = "0.5"` here be removed?

https://github.com/GreptimeTeam/greptimedb/pull/1535/files#diff-6bee95ea4a236cd5427d5462be0f4cdfb9f96df37397f3f0c8d6eff600e0046bR46

Reply:
@MichaelScofield Still need `toml` in the unit tests. I move it to `[dev-dependencies]`.